credsweeper/app.py

```diff
@@ -336,6 +348,21 @@ def data_scan(self, data_provider: DataContentProvider, depth: int, recursive_li
             except Exception as gzip_exc:
                 logger.error(f"{data_provider.file_path}:{gzip_exc}")
 
+        elif data_provider.is_encoded():
```

Comment:
`data_provider.is_encoded()` is good, but we already used `Util.is_gzip()` and `Util.is_zip()` above.
So how about move `is_encoded()` to `Util` class and use it?

Reply:
Util.is_gzip() and Util.is_zip() are functions to recognize the data with short data. They don't make full analysis.
is_encoded() applies full encoding of data. Probably, the name may be changed to avoid a confusion.

Reply:
I meant that the above methods could be used on other class maybe in the future, and `is_xxx()` methods are just check the condition and return the bool result. So suggested to you to put them to `Util` class.